openc3/lib/openc3/utilities/store_queued.rb

```diff
@@ -60,19 +60,29 @@ def initialize(update_interval)
       end
     end
 
+    def set_update_interval(interval)
+      if interval < @update_interval and interval > 0.0
```

Comment:
So you can only lower the interval and not increase it?

Reply:
Correct.